templates/ansible/skeletons/catalog-info/catalog-info.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: backstage.io/v1alpha1
```

Comment:
Shall we keep it within the ansible dir, or move it to the common skeletons dir?

Reply:
Would it be possible to re-use the existing skeleton for catalog-info.yaml? Having one centralized catalog-info template is going to make maintenance much easier.

You can extend existing https://github.com/redhat-developer/red-hat-developer-hub-software-templates/blob/main/skeletons/catalog-info/catalog-info.yaml to add things that you need